src/libraries/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Collections.Generic;
 using System.Linq;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
System.Collections.Generic was already out of order, but can you move this up to be alphabetical?